Client/sdk/core/CDiscordInterface.h

```diff
@@ -0,0 +1,20 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto v1.0
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        sdk/core/CDiscordInterface.h
+ *  PURPOSE:     Discord interface class
+ *
+ *****************************************************************************/
+
+#pragma once
+
+class CDiscordInterface
```

Comment:
Add a virtual destructor for every class that is supposed to be inherited from. There are exceptions to this rule, but the overhead of the virtual dispatch is less than the headaches for any developer down the road.